cli/acp_dac_relationship_add.go

```diff
@@ -87,6 +63,22 @@ Example: Creating a dummy relationship does nothing (from database perspective):
 		},
 	}
 
+	EmbedCLIExample(ctx, cmd, "Let another actor read a private document",
+		`defradb client acp document relationship add \
+	--collection Users \
+	--docID bae-ff3ceb1c-b5c0-5e86-a024-dd1b16a4261c \
+	--relation reader \
+	--actor did:key:z7r8os2G88XXBNBTLj3kFR5rzUJ4VAesbX7PgsA68ak9B5RYcXF5EZEmjRzzinZndPSSwujXb4XKHG6vmKEFG6ZfsfcQn \
+	--identity e3b722906ee4e56368f581cd8b18ab0f48af1ea53e635e3f7b8acd076676f6ac`)
+
+	EmbedCLIExample(ctx, cmd, "Let all actors read a private document",
+		`defradb client acp document relationship add \
+	--collection Users \
+	--docID bae-ff3ceb1c-b5c0-5e86-a024-dd1b16a4261c \
+	--relation reader \
+	--actor "*" \
+	--identity e3b722906ee4e56368f581cd8b18ab0f48af1ea53e635e3f7b8acd076676f6ac`)
```

Comment:
question: Removed the dummy example willingly? or accidental?

Reply:
willingly, I felt it wasn't providing useful context to the user. Happy to put it back in if you feel it should be